site2/docs/tiered-storage-aws.md

```diff
@@ -0,0 +1,283 @@
+---
+id: tiered-storage-aws
+title: Use AWS S3 offloader with Pulsar
+sidebar_label: AWS S3 offloader
+---
+
+This chapter guides you through every step of installing and configuring the AWS S3 offloader and using it with Pulsar.
+
+## Installation
+
+Follow the steps below to install the AWS S3 offloader.
+
+### Prerequisite
+
+- Pulsar: 2.4.2 or later versions
+  
+- Apache jclouds: 2.2.0 or later versions
```

Comment:
Which version of JClouds to use up to the Pulsar, so users could don't care about it.

Reply:
Thanks. Confirmed w/ @gaoran10, users do not care about the Apache jcloud version, so I've deleted this.